src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadPackId.cs

```diff
@@ -43,4 +47,14 @@ public WorkloadPackId(string id)
 
         public static bool operator !=(WorkloadPackId a, WorkloadPackId b) => !a.Equals(b);
     }
+
+#if USE_SYSTEM_TEXT_JSON
+    internal class PackIdJsonConverter : JsonConverter<WorkloadPackId>
+    {
+        public override WorkloadPackId Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) =>
+            new WorkloadPackId(reader.GetString() ?? string.Empty);
+
+        public override void Write(Utf8JsonWriter writer, WorkloadPackId value, JsonSerializerOptions options) => writer.WriteStringValue(value.ToString());
```

Comment:
Is workloadpackID (value parameter) a struct or can it be null?

Reply:
It's a struct defined higher up in this same file: <https://github.com/dotnet/sdk/blob/62fb3746549f754ce45d632f4424a6505ee54d63/src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadPackId.cs#L18>